pkgs/applications/video/animdl/default.nix

```diff
@@ -0,0 +1,70 @@
+{ python3Packages,
+  buildPythonApplication,
+  fetchFromGitHub,
+  lib,
+  poetry-core,
+  anchor-kr,
+  anitopy,
+  click,
+  cssselect,
+  httpx,
+  lxml,
+  packaging,
+  pkginfo,
+  pycryptodomex,
+  pyyaml,
+  regex,
+  rich,
+  tqdm,
+  yarl
+}:
+buildPythonApplication rec {
+  pname = "animdl";
+  version = "1.7.27";
+
+  src = fetchFromGitHub {
+    owner = "justfoolingaround";
+    repo = "animdl";
+    # Using the commit hash because upstream does not have releases. https://github.com/justfoolingaround/animdl/issues/277
+    rev = "c7c3b79198e66695e0bbbc576f9d9b788616957f";
+    hash = "sha256-/XPVWBitFYsUkb9WMlR5F2amAPcVnDtJUgFXP2gXyNk=";
+  };
+
+  format = "pyproject";
```

Comment:
usually we put the format under the version attribute so like:
```
  pname = "animdl";
  version = "1.7.27";
  format = "pyproject";
```
so it would be nice if you move it up.